src/Mono.Android.Runtime/Android.Runtime/AndroidEnvironmentInternal.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+
+namespace Android.Runtime
```

Comment:
…and more name bikeshedding: I think it was a historical mistake to use `Android.Runtime`; we're *lucky* that Google never introduced an `android.runtime` package.

I think `Microsoft.Android.Internals` would be a good namespace, or just `Microsoft.Android`, as all the types here are `internal`.

Reply:
Likewise, I'd rather keep the `Android.Runtime` namespace, because this assembly includes sources from `Mono.Android` that are in this namespace. If we want to rename it, we should rename it all at once one day.